### ISSUE TITLE
no log nor mnesia directories

### ISSUE THREAD

**Issue by pdl86** [gotthardp/lorawan-server#202]
Tsanrd
on a raspbery pi , lorawan installed as a servcie , no log nor mnesia directories ( neither in /var/log nor /usr/lib/lora-wan-server) 
 any idea ? 
regards
Philippe

**Comment by gotthardp** [comment]
If you used the .deb package, the logs should be in `/var/lib/lorawan-server`.

**Comment by pdl86** [comment]
not really, but we have found them in the home directory of the lorawan user ( as  lorawan is started as a service) 
so we close the issue
